styles.css

```diff
@@ -0,0 +1,189 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&family=Open+Sans:wght@400;500;600&family=Poppins:wght@200;300;400;500;600&family=Roboto:wght@400;700;900&display=swap');
+
+:root {
+  --white: #fff;
+  --black: #1c2b2d;
+  --blue: #31326f;
+  --light-blue: #005490;
+  --color-primary: #9d0191;
+  --color-sec: #db6400;
+  --color-grey: #eee;
+  --color-dark-grey: #222831;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.headline {
+  background-color: #1c1a19;
+  margin: 0;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  position: absolute;
+  flex-wrap: nowrap;
+}
+
+.headbar-menu {
+  display: flex;
+  background: #3c3a39;
+  width: 100%;
+  height: 4.5rem;
+  border: 0.125rem solid grey;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
+  padding: 0.625rem 0.75rem;
+  margin-right: 100%;
+  flex-wrap: nowrap;
+}
+
+.logo {
+  padding-inline-start: 2%;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 1em;
+  font-size: 1.125rem;
+  line-height: 1.125rem;
+  align-items: center;
+  text-align: center;
+  color: white;
+  order: 0;
+  margin-left: 0.625rem;
+  text-decoration: none;
+}
+
+.container_text {
+  display: flex;
+  flex-direction: column;
+  background-image: url(PImages/Illustration.png);
+  background-color: #1c1a19;
+  justify-content: space-between;
+  text-align: left;
+  height: auto;
+  width: 100%;
+  padding: 7.75rem 1.5rem;
+  gap: 1rem;
+  background-position: right bottom;
+  background-size: 8.75rem;
+  background-repeat: no-repeat;
+  overflow: hidden;
+  flex: 1;
+  margin-top: 0;
+}
+
+.ptext {
+  color: #ff6b00;
+  font-size: 327px;
+  margin-top: 0%;
+  width: 20.437rem;
+  top: 0.625rem;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  line-height: 3rem;
+}
+
+.stext {
+  font-family: 'Roboto', sans-serif;
+  font-style: bold;
+  font-size: 1.6rem;
+  line-height: 2rem;
+  width: 85%;
+  height: 1.6875rem;
+  text-align: left;
+  align-items: center;
+  color: #fff5e1;
+  margin-top: 0.625rem;
+  margin-bottom: 0.625rem;
+  font-weight: 700;
+}
+
+.description {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 1.25rem;
+  line-height: 1.5rem;
+  color: #fff;
+  margin-top: 0;
+  margin-bottom: 0.8125rem;
+  font-style: normal;
+  width: 100%;
+  text-align: left;
+}
+
+.icon-wrap {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px 151px 120px 24px;
```

Comment:
- [ ] Kindly correct the padding and margin of the social icons to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://github.com/Zuwairanajma/Portfolio-Projects/assets/58642949/b1828c4b-ce5d-4044-aca9-11075d68b607) | ![image](https://github.com/Zuwairanajma/Portfolio-Projects/assets/58642949/3d8cd45b-b046-4515-941b-7303749c2bd1) |